src/typeDefs/types.ts

```diff
@@ -348,6 +348,7 @@ export const types = gql`
     adminApproved: Boolean
     createdAt: DateTime
     tagsAssignedWith(
+      organizationId: ID!
```

Comment:
Make organizationId optional.

Make sure to place arguments in alphabetical order for all new fields you're defining.